src/main/java/edu/kis/powp/jobs2d/command/CommandCounter.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.command;
+
+public class CommandCounter implements CommandVisitor {
+
+	@Override
+	public void visit(OperateToCommand operateToCommand) {
+		System.out.println("Visited OperateToCommand");
+		System.out.println("X: " + operateToCommand.getPosX());
```

Comment:
System.out is not a reusable way of developing components. It is more for logging/debugging or CLI tools. 
Counting should be done using fields in the visitor and the results should be available using getters.